stacks/jenkins-scale.yml

```diff
@@ -24,6 +24,7 @@ services:
           memory: 800M
     extra_hosts:
       - "${SLACK_HOST:-devops20.slack.com}:${SLACK_IP:-54.192.78.227}"
+      - "github.com:54.72.52.58"
```

Comment:
The problem is that IPs are changing all the time and `github.com` with a hard-coded IP is likely going to fail. The line above can be, at least, overwritten with the env. var. `SLACK_HOST`. Also, in all the places where this is mentioned (docs, books, etc.) it is clearly stated that you need to define `SLACK_HOST` and not rely on whatever is the IP put here.